src/DraxList.tsx

```diff
@@ -64,6 +64,8 @@ export const DraxList = <T extends unknown>(
 		onItemReorder,
 		id: idProp,
 		reorderable: reorderableProp,
+		draggable: draggableProp,
```

Comment:
I am not satisfied with the name `draggable`. Based on other Drax components, `draggable` implies to me that I could drag this entire list component around, which I cannot. It's really the individual list items that are draggable. To be more explicit, I think I'll change the name to `itemsDraggable`.